cloud/services/virtualmachines/virtualmachines.go

```diff
@@ -39,6 +41,34 @@ import (
 	"sigs.k8s.io/cluster-api-provider-azure/util/tele"
 )
 
+const (
+	// winAutoLogonFormatString is the format string used to create the AutoLogon
+	// AdditionalUnattendContent configuration for Windows machines.
+	winAutoLogonFormatString = `<AutoLogon>
+			<Username>%s</Username>
+			<Password>
+				<Value>%s</Value>
+			</Password>
+			<Enabled>true</Enabled>
+			<LogonCount>1</LogonCount>
+		</AutoLogon>`
+
+	// winFirstLogonCommandsString is the string used to create the FirstLogonCommands
+	// AdditionalUnattendContent configuration for Windows machines.
+	winFirstLogonCommandsString = `<FirstLogonCommands>
+			<SynchronousCommand>
+				<Description>Copy user data secret contents to init script</Description>
+				<CommandLine>cmd /c "copy C:\AzureData\CustomData.bin C:\init.ps1"</CommandLine>
```

Comment:
per https://github.com/Azure/azure-sdk-for-go/blob/master/services/compute/mgmt/2020-06-01/compute/models.go#L7202-L7203 it seems passing CustomData is not required (if it does, it'll be written to `C:\AzureData\CustomData.bin`).
I reckon these commands here should first check whether the file exists, before copying them (or executing them in the second `SynchronousCommand`. (This check wasn't necessary for us downstream, as our OpenShift machine-api-operator would _always_ pass it iirc)